admin/app/models/certification-center.js

```diff
@@ -21,6 +24,14 @@ export default class CertificationCenter extends Model {
     return types.find((type) => type.value === this.type).label;
   }
 
+  get isArchived() {
+    return Boolean(this.archivedAt);
+  }
+
+  get archivedAtFormatDate() {
+    return this.archivedAt ? dayjs(this.archivedAt).format('DD/MM/YYYY') : null;
```

Comment:
L'opérateur `&&` est ici plus adapté que l'opérateur ternaire : 
```suggestion
    return this.archivedAt && dayjs(this.archivedAt).format('DD/MM/YYYY');
```

Reply:
Mais ce n'est pas le même return, non ? Le second cas renverra false si les conditions ne sont pas remplies, le premier renvoyait null

Reply:
Short-circuit evaluation : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation

Exemple : 
```
> format = s => `ooo${s}ooo`
[Function: format]

> archivedAt = null
null
> archivedAt && format(archivedAt)
null

> archivedAt = '2025'
'2025'
> archivedAt && format(archivedAt)
'ooo2025ooo'
```